### PR TITLE
Feat/env headers

### PR DESCRIPTION
**Pull request by GabrielBrandao1618**
- Add headers field to env
- Add set header command
- Add list headers command as well(still not configured though)

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -231,6 +231,18 @@ pub enum EnvCmd {
...
+        command: HeaderEnvCmd,
+    },
+}
+
+#[derive(Debug, Subcommand)]
+pub enum HeaderEnvCmd {
+    Set { headers: Vec<String> },
+    Ls,
+    Rm(action::env::HeaderRmArgs),
+    Get(action::env::HeaderGetArgs),
```
Comment (EduardoRodriguesF) [inline src/cli.rs:245]: Could we use the same Cmd struct as for endpoint's Header? That would be useful to preserve documentation and the same usage.
The same is true for the `Headers` struct. Having 2 distinct implementations (`endpoint::Headers` and `env::Headers`) is kinda confusing, as they should both implement the same functionality.

This also avoids us needing to reimplementing all the traits.

**Review by EduardoRodriguesF — CHANGES_REQUESTED** [review]

File: src/action/ls.rs [hunk 2]
```diff
@@ -156,6 +156,7 @@ pub fn cmd(ctx: &Ctx, args: Args) {
         }
 
         for child in node.children.iter() {
+            println!("This is a child: {}", child.value.handle());
```
Comment (EduardoRodriguesF) [inline src/action/ls.rs:159]:
```suggestion
```

File: src/cli.rs (hunk 1 above)
Comment (GabrielBrandao1618) [reply src/cli.rs:245]: Yeah, that makes sense!

**Comment by EduardoRodriguesF** [comment]
In code, that is a great implementation now! However, I noticed a tiny bug that at one point was a problem with some other commands.
Listing headers will have double line breaks. Notice the difference in `header ls` to `env header ls`:

image

@GabrielBrandao1618, could you fix that before we merge?

**Comment by EduardoRodriguesF** [comment]
By the way, that probably happens because `Headers` struct's `Display` implementation already prints with line breaks. Try to compare yours with the `header ls` implementation just to make sure.

**Comment by GabrielBrandao1618** [comment]
@EduardoRodriguesF done!

**Review by EduardoRodriguesF — APPROVED** [review]

**Pull request merged**